nyc_taxis/operations/default.json

```diff
@@ -25,6 +25,47 @@
       "retry-until-success": true,
       "include-in-reporting": false
     },
+    {
+      "name": "create-snapshot-repository",
+      "operation-type": "create-snapshot-repository",
+      "repository": "{{ snapshot_repository_name | default('test-repository') }}",
+      "body": {
+        "type": "s3",
+        "settings": {
+          "bucket": "{{ s3_bucket_name }}",
+          "region": "{{ s3_bucket_region }}"
+        }
+      }
+    },
+    {
+      "name": "create-snapshot",
+      "operation-type": "create-snapshot",
+      "repository": "{{ snapshot_repository_name | default('test-repository') }}",
+      "snapshot": "{{ snapshot_name | default('test-snapshot') }}",
+      "body": {
+        "indices": ["nyc_taxis"]
+      }
+    },
+    {
+      "name": "wait-for-snapshot-create",
+      "operation-type": "wait-for-snapshot-create",
+      "repository": "{{ snapshot_repository_name | default('test-repository') }}",
+      "snapshot": "{{ snapshot_name | default('test-snapshot') }}"
+    },
```

Comment:
Suggest renaming this to "wait-for-snapshot-creation" to match the "wait-for-completion" operation below.